bindings/wasm/examples/explorer_util.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+const EXPLORER_URL_BASE = "https://explorer.iota.org/mainnet/transaction";
+
+/*
+  Write out the Targle Explorer URL given the network and message ID, with the given preamble.
+
+  @param {!string} preamble
+  @param {!string} network
+  @param {!string} messageId
+*/
+function logExplorerUrl(preamble, network, messageId) {
+  console.log(`${preamble} ${EXPLORER_URL_BASE}/${network}net/transaction/${messageId}`);
```

Comment:
Is this correct? 

The `EXPLORER_URL_BASE` constant should have the network transaction path:
```
const EXPLORER_URL_BASE = "https://explorer.iota.org/mainnet/transaction";
```

Reply:
Haha this is definitely not correct, good catch! Will update when I have a minute.

Reply:
Decided to inline the prefix because it's not exported anymore.